modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupAnalyticsCloudForm/index.jsx

```diff
@@ -243,49 +243,52 @@ const SetupAnalyticsCloudPage = ({
 						},
 					});
 
-					await Promise.all(
-						analyticsCloud?.incidentReportContact?.map(
-							({email}) => {
-								return client.mutate({
-									context: {
-										displaySuccess: false,
-										type: 'liferay-rest',
-									},
-									mutation: addIncidentReportAnalyticsCloud,
-									variables: {
-										IncidentReportContactAnalyticsCloud: {
-											analyticsCloudWorkspaceId,
-											emailAddress: email,
-											r_accountEntryToIncidentReportContactAC_accountEntryId:
-												project.id,
+					if (!featureFlags.includes('LPS-159127')) {
```

Comment:
Can you do 

if (featureFlags.includes('LPS-159127')) {
 /// blablabla
} else {
 /// blablabla
}

Keep attention in the order

I think is better than

if (!featureFlags.includes('LPS-159127'))

and

if (featureFlags.includes('LPS-159127')) // line 270

Reply:
Thanks for suggestion